DependencyInjection/Configuration.php

```diff
@@ -101,7 +101,7 @@ public function getConfigTreeBuilder()
                                 ->children()
                                     ->scalarNode('label')->end()
                                     ->scalarNode('label_catalogue')->end()
-                                    ->scalarNode('icon')->defaultValue('<i class="fa fa-folder"></i>')->end()
+                                    ->scalarNode('icon')->defaultValue('fa fa-folder')->end()
```

Comment:
Changing the default value is BC break.

Maybe we can introduce a `icon_class` option and deprecate the old one?